source/module_base/parallel_reduce.cpp

```diff
@@ -246,10 +251,12 @@ void Parallel_Reduce::gather_max_double_pool(const int& nproc_in_pool, double& v
 void Parallel_Reduce::gather_min_double_pool(const int& nproc_in_pool, double& v)
 {
 #ifdef __MPI
-    if (nproc_in_pool == 1)
+    if (nproc_in_pool == 1) 
+    {
         return;
+    }
     double* value = new double[nproc_in_pool];
```

Comment:
I would suggest changing 'new' and 'delete' to 'std::vector', and then initialize the vector with 'std::vector<int> v(n, 0);' instead of using Parallel_Reduce::ZEROS function